llrt_core/src/modules/net/mod.rs

```diff
@@ -67,6 +67,21 @@ pub static TLS_CONFIG: Lazy<ClientConfig> = Lazy::new(|| {
         root_certificates.roots.push(cert)
     }
 
+    if let Ok(extra_ca_certs) = env::var(environment::ENV_LLRT_EXTRA_CA_CERTS) {
+        match File::open(&extra_ca_certs) {
+            Ok(file) => {
+                let mut reader = BufReader::new(file);
+                root_certificates.add_parsable_certificates(
+                    rustls_pemfile::certs(&mut reader).filter_map(std::io::Result::ok),
+                );
+            },
+            _ => warn!(
```

Comment:
Is warning sufficient? Isn't this a configuration error?

Reply:
I know Node will just emit a warning whereas Deno will throw a JS error. I suspect throwing a JS error will involve some refactoring since these statics will need to be wrapped in a `Result`. I'll have a look into it 👍

Reply:
I've wrapped the statics in a `Result`. I've used `thiserror` to avoid the usual error boilerplate. Let me know what you think.

I noticed that exceptions thrown within the [init globals](https://github.com/awslabs/llrt/blob/f97d0b92798691458d183d594219c6bbb5406895/llrt_core/src/vm.rs#L426) stage are not manually caught and therefore it just prints `Error: Exception` instead of `Error: [message]`. I'm guessing this isn't intentional so I can raise an issue and fix this separately.